pkgs/applications/graphics/pixinsight/default.nix

```diff
@@ -1,18 +1,60 @@
-{ stdenv, lib, requireFile, wrapQtAppsHook, autoPatchelfHook, makeWrapper, unixtools, fakeroot
-, mailcap, libGL, libpulseaudio, alsa-lib, nss, gd, gst_all_1, nspr, expat, fontconfig
-, dbus, glib, zlib, openssl, libdrm, cups, avahi-compat, xorg, wayland, libudev0-shim
-# Qt 5 subpackages
-, qtbase, qtgamepad, qtserialport, qtserialbus, qtvirtualkeyboard, qtmultimedia, qt3d, mlt
+{ stdenv
+, lib
+, requireFile
+, wrapQtAppsHook
+, autoPatchelfHook
+, makeWrapper
+, unixtools
+, fakeroot
+, mailcap
+, libGL
+, libpulseaudio
+, alsa-lib
+, nss
+, gd
+, gst_all_1
+, nspr
+, expat
+, fontconfig
+, dbus
+, glib
+, zlib
+, openssl
+, libdrm
+, cups
+, avahi-compat
+, xorg
+, wayland
+, libudev0-shim
+, bubblewrap
+, libjpeg8
+, gdk-pixbuf
+, gtk3
+, pango
+  # Qt 6 subpackages
+, qtbase
+, qtserialport
+, qtserialbus
+, qtvirtualkeyboard
+, qtmultimedia
+, qt3d
+, mlt
+, qtlocation
+, qtwebengine
+, qtquick3d
+, qtwayland
+, qtwebview
+, qtscxml
 }:
 
 stdenv.mkDerivation rec {
   pname = "pixinsight";
-  version = "1.8.9-2";
+  version = "1.8.9-3";
 
   src = requireFile rec {
```

Comment:
Instead of using `requireFile`, can we fetch the binary directly?

Reply:
No, it requires a commercial License and login to the software distribution forum.